yt/frontends/gadget/io.py

```diff
@@ -282,7 +285,13 @@ def _identify_fields(self, data_file):
                         kk = j
                         fields.append((ptype, str(kk)))
                 elif (
-                    k in ["Metallicity", "GFM_Metals", "PassiveScalars"]
+                    k
+                    in [
+                        "Metallicity",
+                        "GFM_Metals",
+                        "PassiveScalars",
+                        "GFM_StellarPhotometrics",
+                    ]
```

Comment:
```suggestion
                    in (
                        "Metallicity",
                        "GFM_Metals",
                        "PassiveScalars",
                        "GFM_StellarPhotometrics",
                    )
```

minor suggestion to change this to a tuple as long as we're adding to it